docs/usage_and_configuration/ways_to_install_black.md

```diff
@@ -0,0 +1,46 @@
+# Ways to install Black
+
+There are several ways you can install _Black_.
+
+## Install from PyPI
+
+To install the latest release of _Black_ from [PyPI] (Requires Python 3.8+), use:
+
+`pip install black`
```

Comment:
O - Just thought of it - Please talk about all black's optional installs in this section too, please.

Reply:
Added, please check!